hack/release.sh

```diff
@@ -267,7 +267,7 @@ EOF
 
 	# Add the DEB package to the APT repo
 	DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb
```

Comment:
ok okkkk ill do the other thing

On Wed, May 27, 2015 at 8:56 PM, Tianon Gravi <notifications@github.com>
wrote:

> In hack/release.sh
> <https://github.com/docker/docker/pull/13532#discussion_r31201461>:
>
> > @@ -267,7 +267,7 @@ EOF
> >
> >  	# Add the DEB package to the APT repo
> >  	DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb
>
> and it's taking advantage of a side effect instead of explicitly using a
> glob
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/13532/files#r31201461>.
>